Libraries/TestHelpers.psm1

```diff
@@ -91,6 +91,9 @@ function Upload-RemoteFile($uploadTo, $port, $file, $username, $password, $usePr
 							$username = $args[4];
 							$uploadTo = $args[5];
 							Write-Output "yes" | .\Tools\pscp -v -pw $args[1] -q -P $args[2] $args[3] $username@${uploadTo}: ;
+							if ( $LASTEXITCODE -ne 0 ) {
+								Write-Output "yes" | .\Tools\pscp -v -scp -pw $args[1] -q -P $args[2] $args[3] $username@${uploadTo}: ;
```

Comment:
remove verbose flag from all commands, it produces too much output

Reply:
Verbose is used to collect the log and to check the pattern "Sending file"